samples/MQTT/AwsMQTT/AwsMQTT/app.config

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
nF doesn't support app.config to for assembly binding. Make sure to remove this and rebuild. If VS insists on adding it there is something wrong with one of the referenced dependencies.

Reply:
Done...